composer.json

```
@@ -44,7 +45,22 @@
 		},
 		"bamarni-bin": {
 			"target-directory": "bin/composer"
-		}
+		},
+        "changelogger": {
+			"changelog": "./changelog.txt",
+        	"formatter": "WCAdmin",
```

You should be able to get rid of your `bin/changelogger.php` wrapper if you change this to
```suggestion
        	"formatter": {
        		"filename": "bin/changelogger/WCAdminFormatter.php"
        	},
```

There's also the possibility to do `"formatter": { "class": "WCAdminFormatter" }` if you arrange for the autoloader to be able to find the class (currently it lookes like you have it only look in `src/`).

---

Thank you! I will try that 👍

---

It worked nicely :) Thank you!! I now have only one file 🎉